research/fed-bpt/src/cma_decomposer.py

```diff
@@ -57,6 +57,29 @@ def recompose(self, data: dict, manager: DatumManager = None) -> GaussFullSample
         return instance
 
 
+class CMAOptionsDecomposer(Decomposer):
+    def supported_type(self) -> Type[GaussFullSampler]:
+        return CMAOptions
+
+    def decompose(self, target: CMAOptions, manager: DatumManager = None) -> Any:
+        data = {}
+        data.update(target)
+
+        fields_to_remove = ["CMA_eigenmethod", "CMA_mirrors", "randn", "BoundaryHandler", "is_feasible"]
```

Comment:
where are these names coming from? the cma package?
Can you add a link to documentation that explain these fields?

Reply:
Good point! Added docstrings to explain these classes.